tests/integration/test_fim/test_files/test_audit/test_remove_audit.py

```diff
@@ -63,12 +63,14 @@
 import os
 import re
 import subprocess
+from time import sleep
```

Comment:
This is not used

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3562/commits/742a2366754c9e1896f37968eb0f4eb40d91c916)